src/shared/hooks/useAuth.ts

```diff
@@ -78,6 +78,6 @@ export const useProvideAuth = (): ContextValue => {
     signUp,
     signOut,
     user,
-    didAuthentication,
+    // didAuthentication,
```

Comment:
If you keep this variable, you may use it for early return in src/Pages/Auth/Auth.tsx

Reply:
Yes, but I think it has better to divide responsibilities.